pkg/controller/admissionchecks/provisioning/controller_test.go

```diff
@@ -83,6 +83,7 @@ func requestWithCondition(r *autoscaling.ProvisioningRequest, conditionType stri
 
 func TestReconcile(t *testing.T) {
 	baseWorkload := utiltesting.MakeWorkload("wl", TestNamespace).
+		Active(true).
```

Comment:
By default it's nil, which is interpreted as active. I think it's worth not setting this, in case we are masquerading any null pointer exception.

Reply:
Done